packages/@react-aria/overlays/src/useModalOverlay.ts

```diff
@@ -11,26 +11,15 @@
  */
 
 import {ariaHideOutside} from './ariaHideOutside';
+import {AriaOverlayProps, useOverlay} from './useOverlay';
 import {DOMAttributes} from '@react-types/shared';
 import {mergeProps} from '@react-aria/utils';
 import {OverlayTriggerState} from '@react-stately/overlays';
 import {RefObject, useEffect} from 'react';
-import {useOverlay} from './useOverlay';
 import {useOverlayFocusContain} from './Overlay';
 import {usePreventScroll} from './usePreventScroll';
 
-export interface AriaModalOverlayProps {
-  /**
-   * Whether to close the modal when the user interacts outside it.
-   * @default false
-   */
-  isDismissable?: boolean,
-  /**
-   * Whether pressing the escape key to close the modal should be disabled.
-   * @default false
-   */
-  isKeyboardDismissDisabled?: boolean
-}
+export interface AriaModalOverlayProps extends AriaOverlayProps {}
```

Comment:
We discussed adding `shouldCloseOnInteractOutside` to `usePopover` for submenus today, leaning towards being fine adding it there and for `useModalOverlay` as well as you've done here. I'd personally would prefer just adding `shouldCloseOnInteractOutside` only here instead of all the overlay props until we get use cases for the rest of those props but happy for the rest of the team to weigh in.

Would you mind adding a basic test for this? You can probably add a new test file for useModalOverlay or for the RAC Modal since it extends from AriaModalOverlayProps. https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/overlays/test/useModal.test.js or any of the tests in https://github.com/adobe/react-spectrum/tree/main/packages/react-aria-components/test could serve as a guide